projects/js-packages/connection/package.json

```diff
@@ -1,16 +1,18 @@
 {
 	"name": "@automattic/jetpack-connection",
-	"version": "0.1.1-alpha",
+	"version": "0.2.0-alpha",
 	"description": "Jetpack Connection Component",
 	"author": "Automattic",
 	"license": "GPL-2.0-or-later",
 	"dependencies": {
 		"@wordpress/browserslist-config": "3.0.1",
+		"@wordpress/components": "9.2.6",
```

Comment:
This seems to be an old version of the component. Latest available is `13.0.3`. Is it an option to use the latest available?

Reply:
This seems to be causing some CI failures like https://github.com/Automattic/jetpack/pull/19745/checks?check_run_id=2516520038:

```
[test-adminpage] Error: Cannot find module '@wordpress/components'
```

I guess the `@wordpress/components` dep should be also declared in the root `package.json`? That's what we did in https://github.com/Automattic/jetpack/pull/19134 for the `@wordpress/i18n` dep.

Reply:
Opened https://github.com/Automattic/jetpack/pull/19758 to see if the CI failures get solved.

Reply:
Thanks @mmtr, it looks like your PR solved the issue!
There's a linter failing though, you probably need to run `yarn install` in the project root, then commit updated `yarn.lock`.

Reply:
Hi @jeherve, that's a valid point.
I had to downgrade it at some point to make it match the Jetpack version, otherwise yarn would run into a bunch of non-explainable problems.
We will likely be able to update it separately from Jetpack after switching from Yarn to Node.
Or we could update `@wordpress/components` in both Jetpack and RNA Connection.

Reply:
Added a task to update `@wordpress/components`: 1200295095943283-as-1200307886010305